tutorials/spring-boot-restful/src/main/kotlin/org/jetbrains/kotlin/demo/Application.kt

```diff
@@ -0,0 +1,13 @@
+package org.jetbrains.kotlin.demo
+
+import org.springframework.boot.SpringApplication
+import org.springframework.boot.autoconfigure.SpringBootApplication
+
+@SpringBootApplication
+class Application {
```

Comment:
Indeed, but that's was made on purpose to make more explicit where developers should had there `@Bean` member functions.